README.md

```diff
@@ -32,6 +32,7 @@ https://github.com/conda-forge/miniforge/#download
 
 Create a Conda environment:
 
+    conda config --append channels conda-forge
     conda create -n lp llvmdev=11.0.1 bison=3.4 re2c python cmake make toml numpy
```

Comment:
I think this is better:
```suggestion
    conda create -c conda-forge -n lp llvmdev=11.0.1 bison=3.4 re2c python cmake make toml numpy
```
Note that this is not needed if you install conda from conda-forge, which is what I do. But it does not hurt to have it there explicitly.